### DISCUSSION TITLE
Link Descriptions

### DISCUSSION THREAD

**Discussion by lbollar** [nvim-neorg/neorg#748] [Q&A] [answered]
Hi all, a bit confused, are link descriptions supposed to have some sort of concealer action tied to them.

If I do {https:someurl}[Description] I was expecting concealer or Neorg would make make Description a enterable link and hide all the url, but nothing happens. 

Thanks in advance.

**Comment by lbollar** [comment]
And sorry I missed this discussion https://github.com/nvim-neorg/neorg/discussions/336, that might indicate it isn't an existing feature yet. If that is the case, wondering about any plans to add it in the future.

**Comment by lbollar** [comment]
Also I am seeing this [one](https://github.com/nvim-neorg/neorg/discussions/293), and realizing I don't have any link formatting. Is this something you have to turn on in config? All my local links like {:some-note:} work but they just look like that.

**Comment by miraculusik** [comment, marked as answer]
I guess you are looking this
https://github.com/nvim-neorg/neorg/discussions/753#discussion-4871055

**Reply by miraculusik** [reply]
https://github.com/nvim-neorg/neorg/discussions/753#discussioncomment-5018141

**Reply by lbollar** [reply]
Thank you, I had eventually figured that out. I’m guessing some clearer mentions of this option in concealer docs might help reduce the number of concealer questions like this.

**Reply by miraculusik** [reply]
I agree with you. It may be useful for many people to write a small note for this.

**Reply by max397574** [reply]
@vhyrro
